### ISSUE TITLE
giou_loss = nan

### ISSUE THREAD

**Issue by charliewang789** [hunglc007/tensorflow-yolov4-tflite#194]
hi when I train the model, i got the following, what could be the reason for "nan" value?

=> STEP   40/103560   lr: 0.000006   giou_loss:  nan   conf_loss:  nan   prob_loss:  nan   total_loss:  nan

**Comment by logan-markewich** [comment]
I'm also hitting this. giou loss always hits nan first, then the rest follows on the next step
